examples_tests/13.BRDF_Explorer/BRDFExplorerApp.cpp

```diff
@@ -230,29 +229,12 @@ float ReIoRfromF0andImIoR(float F0, float ImIoR)
             source += "\treturn texture(uMetallicMap, texCoords).x;";
         source += "\n}\n";
 
-        source += "vec3 getIoR_dielectric(in vec2 texCoords) {\n";
-        if (_params.constantRI)
-            source += "\treturn uRealIoR;";
-        else
-            source +=
-            R"(float F0 = texture(uIoRMap, texCoords).x;
-return vec3(IoRfromF0(REFLECTANCE_SCALE_FACTOR*F0*F0));
-)";
-        source += "\n}\n";
-
-        source += "vec3 getIoR_conductor(in vec2 texCoords) {\n";
+        source += "vec3 getReflectance(in vec2 texCoords) {\n";
```

Comment:
you're not returning reflectance, this returns IoR when not using texture?

Reply:
yea, there's a comment in code you commented below

Reply:
so why is it named like that?

Reply:
well I had to name it somehow and choose between the two